File: corte1 /Python/Constructor/MainRectangulo.py
class Rectangulo:
    def __init__(self, b, h):
        self.base = b
        self.altura = h
        
    def area(self):
        return self.base * self.altura

r1 = Rectangulo(3.0, 2.0)

print("Base =", r1.base)
print("Altura =", r1.altura)
print("Area =", r1.area())

File: corte1 /Java/Rectangulo/src/MainRectangulo.java
class Rectangulo {
    public double base;
    public double altura;

    public Rectangulo(double b, double h) {
        base = b;
        altura = h;
 }



    public double area() {
    return base * altura;
    }
}

public class MainRectangulo {
    public static void main(String[] args) {
        Rectangulo r1 = new Rectangulo(3.0, 2.0);

System.out.println("Base = " + r1.base);
System.out.println("Altura = " + r1.altura);
System.out.println("Area = " + r1.area());
    }
}